### PR TITLE
PNSE BinaryFormatter on mobile and skip related tests

### PR DESCRIPTION
**Pull request by steveisok**

### CODE REVIEW

File: src/libraries/System.Runtime.Serialization.Formatters/src/System.Runtime.Serialization.Formatters.csproj [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-    <NoWarn Condition="'$(TargetsBrowser)' == 'true'">$(NoWarn);CS0649</NoWarn>
+    <NoWarn Condition="'$(TargetsMobile)' == 'true'">$(NoWarn);CS0649</NoWarn>
   </PropertyGroup>
   <!-- ILLinker settings -->
   <PropertyGroup>
     <ILLinkDirectory>$(MSBuildThisFileDirectory)ILLink\</ILLinkDirectory>
   </PropertyGroup>
-  <ItemGroup Condition="'$(TargetsBrowser)' != 'true'">
+  <ItemGroup Condition="'$(TargetsMobile)' != 'true'">
     <ILLinkSubstitutionsXmls Include="$(ILLinkDirectory)ILLink.Substitutions.NonBrowser.xml" />
```
Comment (MaximLipnin) [inline src/libraries/System.Runtime.Serialization.Formatters/src/System.Runtime.Serialization.Formatters.csproj:13]: Do we need to rename `ILLink.Substitutions.NonBrowser.xml` as well?

**Review by MaximLipnin — APPROVED** [review]

File: src/libraries/System.Runtime.Serialization.Formatters/src/System.Runtime.Serialization.Formatters.csproj (hunk 1 above)
Comment (steveisok) [reply src/libraries/System.Runtime.Serialization.Formatters/src/System.Runtime.Serialization.Formatters.csproj:13]: Yeah, it would make sense.

**Review by GrabYourPitchforks — APPROVED** [review]
LGTM! Feel free to change the filename if you wish per the other comment, I don't really have a strong preference one way or another.

**Review by radical — APPROVED** [review]

File: src/tasks/AndroidAppBuilder/AndroidAppBuilder.cs [hunk 2]
```diff
@@ -69,6 +69,16 @@ public class AndroidAppBuilderTask : Task
 
     public bool ForceInterpreter { get; set; }
 
+    /// <summary>
+    /// Forces the runtime to use the invariant mode
+    /// </summary>
+    public bool InvariantGlobalization { get; set; }
```
Comment (marek-safar) [inline src/tasks/AndroidAppBuilder/AndroidAppBuilder.cs:75]: This should not be needed when we have runtimeconfig.json support

**Review by akoeplinger — APPROVED** [review]

**Pull request merged**